docs/ref/contrib/postgres/fields.txt

```diff
@@ -222,7 +222,7 @@ lookups available after the transform do not change. For example::
     >>> Post.objects.create(name='Third post', tags=['django', 'python', 'thoughts'])
 
     >>> Post.objects.filter(tags__0_1=['thoughts'])
-    <QuerySet [<Post: First post>]>
+    <QuerySet [<Post: First post>, <Post: Second post>]>
 
     >>> Post.objects.filter(tags__0_2__contains='thoughts')
```

Comment:
I think that this example can be fixed by surrounding the 'thoughts' argument with square brackets:
```
>>> Post.objects.filter(tags__0_2__contains=['thoughts'])
```

I've updated the PR accordingly.